data/data/aws/vpc/variables.tf

```diff
@@ -11,14 +11,10 @@ variable "cluster_id" {
   type = string
 }
 
-variable "private_master_endpoints" {
-  description = "If set to true, private-facing ingress resources are created."
-  default     = true
-}
-
-variable "public_master_endpoints" {
-  description = "If set to true, public-facing ingress resources are created."
-  default     = true
+variable "publish_strategy" {
+  type        = string
+  default     = "External"
+  description = "The publishing strategy for endpoints like Load balancers"
```

Comment:
nit: "Load" -> "load".  Or, if you go AWS-proper-name, "Load Balancers".

Reply:
fixed with https://github.com/openshift/installer/compare/586800a7998182fd87bc00700adf24a49df216be...83adc04d6cf70ffc2f45bbe6a5e8d1d06173832d